api/devicerepository.proto

```diff
@@ -340,14 +340,26 @@ message ListEndDeviceModelsResponse {
 message EncodedMessagePayload {
   uint32 f_port = 1 [(validate.rules).uint32.lte = 255];
   bytes frm_payload = 2;
-  repeated string warnings = 3 [(validate.rules).repeated.max_items = 10, (validate.rules).repeated.items.string.max_len = 100];
-  repeated string errors = 4 [(validate.rules).repeated.max_items = 10, (validate.rules).repeated.items.string.max_len = 100];
+  repeated string warnings = 3 [(validate.rules).repeated = {
```

Comment:
Actually where do these come from?

This is not defined in the Device Repository.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/3772#discussion_r622073038
https://github.com/TheThingsNetwork/lorawan-stack/pull/3772#pullrequestreview-647998234

The message that contains `EncodedMessagePayload`, `MessagePayloadEncoder`, is still a response message only.